services/node_update.proto

```diff
@@ -148,4 +149,14 @@ message NodeUpdateTransactionBody {
      * If set, the new value SHALL replace the existing hash value.
      */
     google.protobuf.BytesValue grpc_certificate_hash = 7;
+
+    /**
+    * An administrative key controlled by the node operator.
+     * <p>
+     * This key MUST sign this transaction.<br/>
+     * This key MUST sign each transaction to update this node.<br/>
+     * This field MUST contain a valid `Key` value.<br/>
+     * This field is REQUIRED and MUST NOT be set to an empty `KeyList`.
```

Comment:
Slight error here (sorry for that).  This field is not actually _required_ in this transaction.
```suggestion
     * This field is OPTIONAL.<br/>
     * If set, this key MUST sign this transaction.<br/>
     * If set, this key MUST sign each transaction to update this node.<br/>
     * If set, this field MUST contain a valid `Key` value.<br/>
     * If set, this field MUST NOT be set to an empty `KeyList`.
```